swagger_parser/test/parser/data_classes_test.dart

```diff
@@ -10,7 +10,7 @@ void main() {
           p.join('test', 'parser', 'schema', 'empty_class.2.0.json');
       final configFile = schemaFile(schemaPath);
       final schemaContent = configFile!.readAsStringSync();
-      final parser = OpenApiParser(ParserConfig(schemaContent, isJson: true));
```

Comment:
Do not change the input parameters, it is more correct to change the expectedDataClass.